go.mod

```diff
@@ -70,7 +68,7 @@ require (
 	github.com/hashicorp/terraform-exec v0.21.0
 	github.com/in-toto/in-toto-golang v0.9.0
 	github.com/knqyf263/go-apk-version v0.0.0-20200609155635-041fdbb8563f
-	github.com/knqyf263/go-deb-version v0.0.0-20230223133812-3ed183d23422
+	github.com/knqyf263/go-deb-version v0.0.0-20241115131756-1eadde5a9b91
```

Comment:
Can we use https://github.com/knqyf263/go-deb-version/commit/6f4aee6ccd23ac99bc42d410aa85fa407b8f7ef4 commit?

Reply:
I meant to point to the commit. Thanks.
https://github.com/aquasecurity/trivy/pull/7928/commits/b9be8a80a5a8b8878d5bbe5b38c2a81817bf456b